lmp/jobserv.yml

```diff
@@ -271,8 +276,6 @@ triggers:
         params:
           IMAGE: lmp-base-console-image
           MFGTOOL_FLASH_IMAGE: lmp-base-console-image
-          EULA_stm32mp1eval: "1"
-          EULA_stm32mp1disco: "1"
```

Comment:
I prefer if you could create a commit only for this change. This way we can see only EULA changes and revert if needed